docs/source/additional/privacy-strategy.rst

```diff
@@ -0,0 +1,133 @@
+How Substra thinks about privacy.
+=================================
+
+.. _Privacy Strategy:
+
+As proponents of Privacy Enhancing Technologies (PETs), the maintainers of Substra believe it is essential for us to outline our stance on these technologies and provide a comprehensive view of our current understanding of the field.
+
+Here we elaborate on what PETs are and provide a quick summary of the most popular ones. We then take a look at the different types of attacks that can result in data leakage or data theft and then also propose best practices for project governance and security in order to mitigate the risks of potential attacks. We also elaborate on the collaboration required between different personas to ensure data integrity and safe handling of data. 
+
+As pointed out by Katharine Jarmul in her (primer on Privacy Enhancing Technologies)[https://martinfowler.com/articles/intro-pet.html], “Privacy is a technical, legal, political, social and individual concept.” As such, technical solutions are an important part of the answer, but they must be used in conjunction with good data governance and reliable security measures.
```

Comment:
```suggestion
As pointed out by Katharine Jarmul in her [primer on Privacy Enhancing Technologies](https://martinfowler.com/articles/intro-pet.html), “Privacy is a technical, legal, political, social and individual concept.” As such, technical solutions are an important part of the answer, but they must be used in conjunction with good data governance and reliable security measures.
```

Reply:
Well, of course, it's RST 🤦 
So we need the stupid syntax for external URLS:

`primer on Privacy Enhancing Technologies <https://martinfowler.com/articles/intro-pet.html>`_

Reply:
(yes, backticks + a trailing underscore)